cmd/onboard.go

```diff
@@ -226,49 +238,104 @@ TO1:
 		case protocol.HTTPSTransport:
 			scheme, port = "https://", "443"
 		default:
-			slog.Error("Error: Invalid transport protocol", "transport protocol", to2Addr.TransportProtocol)
-			to2URLsErrors = append(to2URLsErrors, fmt.Errorf("invalid transport protocol: %s", to2Addr.TransportProtocol))
+			slog.Error("Unsupported transport protocol", "transport protocol", to2Addr.TransportProtocol)
 			continue
 		}
 		if to2Addr.Port != 0 {
 			port = strconv.Itoa(int(to2Addr.Port))
 		}
 
 		// Check and add DNS address if valid and resolvable
-		if to2Addr.DNSAddress != nil && isResolvableDNS(*to2Addr.DNSAddress) {
-			host := *to2Addr.DNSAddress
-			to2URLs = append(to2URLs, scheme+net.JoinHostPort(host, port))
+		if to2Addr.DNSAddress != nil {
+			if isResolvableDNS(*to2Addr.DNSAddress) {
+				host := *to2Addr.DNSAddress
+				ownerURLs = append(ownerURLs, scheme+net.JoinHostPort(host, port))
+			} else {
+				slog.Warn("DNS address is not resolvable", "dns", *to2Addr.DNSAddress)
+			}
 		}
 
 		// Check and add IP address if valid
-		if to2Addr.IPAddress != nil && isValidIP(to2Addr.IPAddress.String()) {
-			host := to2Addr.IPAddress.String()
-			to2URLs = append(to2URLs, scheme+net.JoinHostPort(host, port))
+		if to2Addr.IPAddress != nil {
+			if isValidIP(to2Addr.IPAddress.String()) {
+				host := to2Addr.IPAddress.String()
+				ownerURLs = append(ownerURLs, scheme+net.JoinHostPort(host, port))
+			} else {
+				slog.Warn("IP address is not valid", "ip", to2Addr.IPAddress.String())
+			}
 		}
 	}
 
-	if len(to2URLs) == 0 {
-		return nil, errors.Join(to2URLsErrors...)
+	// Check if TO1 succeeded but returned no valid TO2 addresses
+	if len(ownerURLs) == 0 {
+		slog.Error("TO1 succeeded but no valid TO2 addresses found")
```

Comment:
This shouldn't be an error, because it's entirely possible to handle it (delay as requested and continue). However, it's unexpected... Either WARN or INFO is fine. Probably INFO, since there's nothing the user can act on - the manufacturer programmed the device weird.